backend/controllers/accounts.js

```diff
@@ -1,27 +1,67 @@
 import { StatusCodes } from 'http-status-codes';
 import accountsService from '../services/accounts.js';
 
-export const getAll = (req, res) => {
-  const accounts = accountsService.getAccounts();
-  res.status(StatusCodes.OK).json(accounts);
+export const getAll = async (req, res) => {
+  try {
+    const accounts = await accountsService.getAccounts(req.user.id);
+    res.status(StatusCodes.OK).json(accounts);
+  } catch (error) {
+    res
```

Comment:
It's better to move this logic to separate method (f.e. handlError(res, error)) to avoid code duplicate